config/samples/machines.yaml

```diff
@@ -0,0 +1,18 @@
+items:
+  - apiVersion: "cluster.k8s.io/v1alpha1"
+    kind: Machine
+    metadata:
+      generateName: aws-controlplane-
```

Comment:
Does `kustomize` also add the name prefix when `generateName` is used in place of name?